pkg/apis/druid/v1alpha1/druid_types.go

```diff
@@ -185,6 +187,9 @@ type DruidNodeSpec struct {
 	// Optional
 	ReadinessProbe *v1.Probe `json:"readinessProbe,omitempty"`
 
+	// Optional
+	AutoScaler *autoscalev2beta1.HorizontalPodAutoscalerSpec `json:"autoscaler,omitempty"`
```

Comment:
I noticed there is a thing  called "Vertical Pod Autoscaler" (in early development) as well, so it might  make more sense to call change this to...
```suggestion
	HPAutoScaler *autoscalev2beta1.HorizontalPodAutoscalerSpec `json:"hpAutoscaler,omitempty"`
```

so that, in future, we could add a `VPAutoScaler` too.

Reply:
valid agreed